app/src/main/res/layout/dialog_image_preview.xml

```diff
@@ -0,0 +1,143 @@
+                android:layout_height="446dp"
+                android:contentDescription="@null"
+                app:actualImageScaleType="fitXY" />
+
+            <LinearLayout
+                android:id="@+id/detailsHolder"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="vertical" />
+
+            <LinearLayout
+                android:id="@+id/moreInfoContainer"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginEnd="16dp"
+                android:layout_marginBottom="16dp"
+                android:orientation="vertical">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="48dp"
+                    android:fontFamily="sans-serif"
+                    android:gravity="center_vertical"
+                    android:letterSpacing="0.04"
+                    android:lineSpacingExtra="6sp"
+                    android:text="@string/suggested_edits_image_more_info"
+                    android:textAllCaps="true"
+                    android:textColor="?attr/primary_text_color"
+                    android:textSize="14sp"
+                    android:textStyle="bold"
+                    tools:text="IMAGE CAPTION" />
+
+                <View
+                    android:layout_width="match_parent"
+                    android:layout_height="1dp"
+                    android:background="?attr/chip_background_color" />
+
+                <LinearLayout
+                    android:id="@+id/imagePageCommonsLinkContainer"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content">
+
+                    <TextView
+                        android:id="@+id/detailTextView"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="11dp"
+                        android:fontFamily="sans-serif"
+                        android:lineSpacingExtra="8sp"
+                        android:text="@string/suggested_edits_image_file_page_link_text"
+                        android:textColor="?attr/colorAccent"
+                        android:textSize="16sp"
+                        android:textStyle="normal" />
+
+                    <ImageView
+                        android:layout_width="@dimen/bottom_toolbar_icon_size"
+                        android:layout_height="@dimen/bottom_toolbar_icon_size"
+                        android:layout_gravity="center_vertical"
+                        android:contentDescription="@null"
+                        android:padding="12dp"
+                        android:tint="?attr/main_toolbar_icon_color"
```

Comment:
Should the `tint` be `?attr/colorAccent`?

Reply:
Great catch!

Reply:
Has it been fixed? I am still seeing the incorrect tint color.